sdks/java/core/src/test/java/org/apache/beam/sdk/PipelineTest.java

```diff
@@ -225,7 +225,7 @@ public void testPipelineSDKExceptionHandling() {
   @Test
   @Category(ValidatesRunner.class)
   public void testMultipleApply() {
-    PTransform<PCollection<? extends String>, PCollection<String>> myTransform = addSuffix("+");
+    MapElements<String, String> myTransform = addSuffix("+");
```

Comment:
Suggest changing `? extends String` to just `String` but otherwise leaving the type as `PTransform`. This is a good way to indicate that you don't intend to use any other facts about the transform.

Reply:
Yes I agree that the PTransform form is more abstract so better, but because of the capricious Java bounds checking this is not possible without doing the `<? extends String>` so I prefer to revert this change and let only  the JsonToRow one, which is the one I care about (and the real goal of this PR).
Thanks for the review Kenn.